Source/PLCrashReporter.h

```diff
@@ -134,4 +137,7 @@ typedef struct PLCrashReporterCallbacks {
 
 - (void) setCrashCallbacks: (PLCrashReporterCallbacks *) callbacks;
 
+- (NSString *) getUserInfo;
+- (void) setUserInfo: (NSString *) userInfo;
```

Comment:
`getUserInfo` -> `userInfo`

Also, it looks like just `@property` will work fine here

Reply:
Changed to a property with name customData